dev/io.openliberty.xmlws.3.0.internal.tools/src/com/ibm/ws/jaxws/tools/WsImport.java

```diff
@@ -0,0 +1,86 @@
+/*******************************************************************************
+ * Copyright (c) 2020 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.jaxws.tools;
+
+import java.io.File;
+import java.io.PrintStream;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+import java.util.Arrays;
+
+import com.ibm.ws.jaxws.tools.internal.JaxWsToolsConstants;
+import com.ibm.ws.jaxws.tools.internal.JaxWsToolsUtil;
+import com.sun.tools.ws.wscompile.WsimportTool;
+
+/**
+ * IBM Wrapper for wsimport tool.
+ */
+public class WsImport {
+    private static final PrintStream err = System.err;
+
+    public static void main(String[] args) {
+        if (WsToolsUtils.isTargetRequired(args)) {
+            String errMsg = JaxWsToolsUtil.formatMessage(JaxWsToolsConstants.ERROR_PARAMETER_TARGET_MISSED_KEY);
+            err.println(errMsg);
+            System.exit(1);
+        }
+        AccessController.doPrivileged(new PrivilegedAction<Object>() {
+            @Override
+            public Object run() {
+                System.setProperty("jakarta.xml.accessExternalSchema", "all");
+                return null;
+            }
+        });
+        //Pass in the JWS and JAX-B APIs as a -classpath arg when Java 9 or above.
+        //Otherwise the javac process started by the tooling doesn't contain these APIs.
+        if (WsToolsUtils.getMajorJavaVersion() > 8) {
```

Comment:
For xmlWS 3.0 none of the APIs will be part of the JDK so this if statement doesn't make sense.